testsuite/tests/backtrace/pr2195.ml

```diff
@@ -0,0 +1,18 @@
+(* TEST
+   flags += "-g"
+   ocamlrunparam = "b"
```

Comment:
@shindere, this idiom is used in other places apparently with the intention to cause `OCAMLRUNPARAM` to be set, but it doesn't seem to. Is it supposed to?